addon/services/unified-event-handler.js

```diff
@@ -122,8 +122,8 @@ export default Ember.Service.extend(Ember.Evented, {
       };
 
       if (!targetHandlers) {
-        this[_HANDLER_MAP][target] = Object.create(null);
-        this[_HANDLER_MAP][target][eventName] = handlerInfo;
+        handlerMap[target] = targetHandlers = Object.create(null);
+        targetHandlers[eventName] = handlerInfo;
```

Comment:
Since this is the same as the `else` block below. Can we eliminate the `else` and just have this on one line?